.gitignore

```diff
@@ -25,3 +25,6 @@ kolibri-design-system.iml
 # env
 .envrc
 .python-version
+
+# local
+myenv
```

Comment:
Is this an env file to save the `PERCY_TOKEN` env variable right?

Reply:
I'm sorry I forgot to remove it during the cleanup. Will do that in the next set of changes